orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/RestrictExecutionDuringTimeWindowSpec.groovy

```diff
@@ -27,24 +29,29 @@ import spock.lang.Specification
 import spock.lang.Subject
 import spock.lang.Unroll
 import static com.netflix.spinnaker.orca.pipeline.RestrictExecutionDuringTimeWindow.SuspendExecutionDuringTimeWindowTask
-import static com.netflix.spinnaker.orca.pipeline.RestrictExecutionDuringTimeWindow.SuspendExecutionDuringTimeWindowTask.TimeWindow
+import static com.netflix.spinnaker.orca.pipeline.RestrictExecutionDuringTimeWindow.TimeWindow
 
 @Unroll
 class RestrictExecutionDuringTimeWindowSpec extends Specification {
```

Comment:
Looks like the name of this class should change since it's testing a different `@Subject`.